packages/tslint-config/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
     "extends": "../../tsconfig",
     "compilerOptions": {
-        "typeRoots": ["../../node_modules/@types", "node_modules/@types"],
-        "outDir": "lib"
+        "outDir": "lib",
```

Comment:
`typeRoots` is already specified in the base __tsconfig.json__ file. We don't need to specify it again.